pensjon-brevbaker/gradle.properties

```diff
@@ -0,0 +1 @@
+ksp.incremental.intermodule=true
```

Comment:
Denne featuren er experimental, går det greit? https://kotlinlang.org/docs/ksp-incremental.html#aggregating-vs-isolating

Reply:
Jeg tenker at det er greit når vi snakker om noe som generer kode for oss, men ikke en eksperimentell funksjonalitet i f.eks brevbaker ktor. Den er ganske nyttig også, om den fungerer som advertert. Uten den så vil alle selectors genereres på nytt når man oppgraderer f.eks api-model, men med den den så burde den klare å identifisere hvilke selectors som er påvirket.